### PR TITLE
Enhance ga and use new custom dimensions

### PR DESCRIPTION
**Pull request by AWare**
Enhancements to our ga tracker that should let us integrate react with it better. 
(todo on ga: add ecommerce tracker)
Bonus: dead code deletion.

### CODE REVIEW

**Comment by AWare** [comment]
Moving this to a WIP and pinging @ajosephides

**Comment by AWare** [comment]
@ajosephides, @paulbrown1982 Here's the new GA tracking for contributions. If this is okay, I'm going to add this into membership too.

**Comment by paulbrown1982** [comment]
👍  and +1 for removing the Membership tier and ecommerce stuff.

**Comment by desbo** [comment]
looks good to me 👍

**Comment by ajosephides** [comment]
👍 - Looks good to me and happy to ship the same to Membership.
I'll get a filtered view together on GA for contribute

File: assets/javascripts/src/modules/analytics/ga.es6 [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    //Set the custom dimensions.
+    let u = user.getUserFromCookie();
+    let isLoggedIn = !!u;
+    let signedOut = !!cookie.getCookie('GU_SO') && !isLoggedIn;
+    ga('set', dimensions.signedIn, isLoggedIn.toString());
+    ga('set', dimensions.isLoggedOn, isLoggedIn.toString());
+    ga('set', dimensions.signedOut, signedOut.toString());
+    ga('set', dimensions.ophanBrowserId, cookie.getCookie('bwid'));
+    if ("ophan" in window.guardian) {
```
Comment (paulbrown1982) [inline assets/javascripts/src/modules/analytics/ga.es6:52]: Should check guardian.ophan exists as it gets inserted asynchronously

File: assets/javascripts/src/modules/analytics/ga.es6 [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+    //Set the custom dimensions.
+    let u = user.getUserFromCookie();
+    let isLoggedIn = !!u;
+    let signedOut = !!cookie.getCookie('GU_SO') && !isLoggedIn;
+    ga('set', dimensions.signedIn, isLoggedIn.toString());
+    ga('set', dimensions.isLoggedOn, isLoggedIn.toString());
+    ga('set', dimensions.signedOut, signedOut.toString());
+    ga('set', dimensions.ophanBrowserId, cookie.getCookie('bwid'));
```
Comment (paulbrown1982) [inline assets/javascripts/src/modules/analytics/ga.es6:51]: cookie.getCookie('bwid')); returns null, that'll send "null" to GA, so this should be wrapped like loggedIn

**Pull request merged**